packages/combobox/src/utils/ComboboxTestUtils.tsx

```diff
@@ -199,8 +199,12 @@ export function renderCombobox<T extends Select>(
    * @param newProps
    * @returns
    */
-  const rerenderCombobox = (newProps: renderComboboxProps) =>
-    renderResult.rerender(getComboboxJSX({ ...props, ...newProps }));
+  const rerenderCombobox = (newProps: renderComboboxProps) => {
+    const rerenderProps = { ...props, ...newProps };
+    return renderResult.rerender(
+      getComboboxJSX(rerenderProps as renderComboboxProps),
```

Comment:
We should type the variable not the argument
```ts
const rerenderProps: renderComboboxProps = { ...props, ...newProps };
```

Reply:
we have to cast it, TS still raises an error when we type the variable

Reply:
Hmm, I still think asserting at assignment is technically better
```ts
const rerenderProps = {
    ...props,
    ...newProps,
} as renderComboboxProps;
```